app/models/spree/shipment_decorator.rb

```diff
@@ -40,19 +40,28 @@ def rebuild_easypost_shipment
       new_ep_rates = new_ep_shipment.rates
 
       matching_rates = new_ep_rates.select do |sr|
-        ("#{sr.carrier} #{sr.service}" == selected_shipping_rate.name) &&
-          (sr.rate.to_f == selected_shipping_rate.cost) &&
-          (selected_shipping_rate.shipping_method.code.match(/#{sr.carrier_account_id}/))
+        names_match = "#{sr.carrier} #{sr.service}" == selected_shipping_rate.name
```

Comment:
preference. could we rename sr to shipping_rate?

Reply:
I went with `ep_shipping_rate`, since it's not the Spree::ShippingRate model, but in fact EasyPost::Shipment#rates